pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Ben Memberg <benmemberg@gmail.com>", "Connor Rubin <csrubin@gmail.co
 python = "^3.10"
 nicegui = "^1.2.24"
 pywebview = "^4.2.2"
+sqlalchemy = "^2.0.23"
```

Comment:
Btw, I had issues trying to install `psycopg2` via poetry (which will be needed to work with Postgres). Workaround is to use `poetry add psycopg2-binary`